db/migrations/versions/009_postcode_as_array.py

```diff
@@ -0,0 +1,35 @@
+"""empty message
+
+Revision ID: 009_postcode_as_array
+Revises: 008_add_new_round_4_cols
+Create Date: 2023-11-16 17:22:58.496260
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "009_postcode_as_array"
+down_revision = "008_add_new_round_4_cols"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("project_dim", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("postcode_list", sa.ARRAY(sa.VARCHAR()), nullable=True))
+    op.execute("UPDATE project_dim SET postcode_list = STRING_TO_ARRAY(postcodes, ',')")
```

Comment:
I'm wondering, and this might be crazy, but could we simplify the whole thing by keeping ingest / validation as it is, and just executing STRING_TO_ARRAY on the value at ingest time for this particular column?

Reply:
yeah, not needed now you've extended the serialiser. I'll close this